app/domain/authentication/authn_jwt/restriction_validation/create_constraints_from_policy.rb

```diff
@@ -0,0 +1,104 @@
+require 'command_class'
+
+module Authentication
+  module AuthnJwt
+    module RestrictionValidation
+      CreateConstrainsFromPolicy = CommandClass.new(
+        dependencies: {
+          non_permitted_constraint_class: Authentication::Constraints::NonPermittedConstraint,
+          required_constraint_class: Authentication::Constraints::RequiredConstraint,
+          multiple_constraint_class: Authentication::Constraints::MultipleConstraint,
+          not_empty_constraint: Authentication::Constraints::NotEmptyConstraint.new,
+          fetch_mandatory_claims: Authentication::AuthnJwt::RestrictionValidation::FetchMandatoryClaims.new,
+          fetch_mapping_claims: Authentication::AuthnJwt::RestrictionValidation::FetchMappingClaims.new,
+          logger: Rails.logger
+        },
+        inputs: %i[authentication_parameters non_permitted_constraints]
+      ) do
+        def call
+          @logger.debug(LogMessages::Authentication::AuthnJwt::CreateContraintsFromPolicy.new)
+          init_constraints_list
+          add_non_empty_constraint
+          fetch_mandatory_claims
+          fetch_claims_mapping
+          map_mandatory_claims
+          add_required_constraint
+          add_non_permitted_constraint
+          @logger.debug(LogMessages::Authentication::AuthnJwt::CreatedConstraintsFromPolicy.new)
+          create_multiple_constraint
+        end
+
+        private
+
+        def init_constraints_list
+          @constraints = []
+        end
+
+        def add_non_empty_constraint
+          @constraints.append(@not_empty_constraint)
+        end
+
+        def load_mandatory_claims
+          mandatory_claims
+        end
+
+        def fetch_mandatory_claims
+          mandatory_claims
+        end
+
+        def map_mandatory_claims
```

Comment:
`fetch_mandatory_claims` and `map_mandatory_claims` are in-consist please align them:
Or change  `fetch_mandatory_claims` function to @fetched_mandatory_claims = ...
Or change  `map_mandatory_claims` to contains another function called `mapped_mandatory_claims`